packages/gnome_tweaks.rb

```diff
@@ -23,4 +23,6 @@ class Gnome_tweaks < Meson
   depends_on 'python3'
   depends_on 'libhandy'
   depends_on 'libnotify'
```

Comment:
Update to [45.1](https://gitlab.gnome.org/GNOME/gnome-tweaks/-/tags/45.1)?

Reply:
Requires deprecated gnome_desktop_daemon, deprecating.